dashboard_app/view_mixins.py

```diff
@@ -1,6 +1,6 @@
 """Useful mixins for views."""
 from django import http
-from django.utils import simplejson as json
+import json
```

Comment:
We have imports in the following scheme:

```
1. builtins
2. django imports
3. third party imports
4. project internal imports
```

So please move this line to the top:

```
import json

from django import http
...
```